### DISCUSSION TITLE
Alf.io Build Fails - HELP

### DISCUSSION THREAD

**Discussion by iansiple1025** [alfio-event/alf.io#1432] [General]
I am running into an issue where every time I build, it fails and gives me these errors:

Caused by:
        org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'controllerConfiguration': Unsatisfied dependency expressed through field 'objectMapper': Error creating bean with name 'alfio.BaseTestConfiguration': Invocation of init method failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'springSessionRepositoryFilter' defined in org.springframework.session.config.annotation.web.http.SpringHttpSessionConfiguration: Unsatisfied dependency expressed through method 'springSessionRepositoryFilter' parameter 0: Error creating bean with name 'migrator' defined in alfio.config.DataSourceConfiguration: Unsatisfied dependency expressed through method 'migrator' parameter 0: Error creating bean with name 'getDataSource' defined in alfio.config.DataSourceConfiguration: Failed to instantiate [javax.sql.DataSource]: Factory method 'getDataSource' threw exception with message: Required key 'datasource.url' not found

Any help would be greatly appreciated.

Thank you!

**Comment by cbellone** [comment]
Hi, please put the entire stack trace in a [gist](https://gist.github.com) and add the link here.
In addition to that, please double check that you have docker available on your machine by running:
`docker ps`
